mkdocs.yml

```diff
@@ -209,7 +209,10 @@ nav:
       - 'Course configuration': 'course/index.md'
       - 'Course instance configuration': 'courseInstance.md'
       - 'Assessment configuration': 'assessment/index.md'
-      - 'Question configuration': 'question.md'
+      - 'Question configuration':
+          - 'question/index.md'
+          - '<code>question.html</code>': 'question/html.md'
+          - '<code>server.py</code>': 'question/server.md'
       - 'Question runtime environment': 'questionRuntime/index.md'
       - 'Elements for writing questions': 'elements.md'
```

Comment:
I think you could add the elements under "Question configuration" as well, but I'd prefer to hear from @nwalters512 or @mwest1066 about that.

Reply:
I somewhat like that idea. If we do that, maybe "Question configuration" should be "Question authoring" instead? Let's get input from others before we make any changes here (and maybe save that for a future PR anyways).

Reply:
Agree on everything that @nwalters512 said!